portfolio/src/main/webapp/index.html

```diff
@@ -10,7 +10,24 @@
     <div id="content">
       <h1>Elijah's Portfolio</h1>
       <h4>Bio</h4>
+       <p> I was born and raised in Chicago, Illinois. I currently live in Logan Square,
+           and I study Computer Science at the Univerity of Illinois at Chicago. In my 
+           free time I enjoy biking,dancing, and reading comic books. 
+       </p>
       <h4>Projects</h4>
+        <ul>
+            <li>Research: Utilized natural language processing and Naive Bayes classifiers 
```

Comment:
Indentations should be 2 spaces to conform to the [style guide](https://google.github.io/styleguide/htmlcssguide.html#Indentation).

This lets a reader easily distinguish between indentations and [line-wrapping](https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping).